docs/TektonPipeline.md

```diff
@@ -57,6 +57,15 @@ spec:
     threads-per-controller: 2
     kube-api-qps: 5.0
     kube-api-burst: 10
+  options:
+    disabled: false
+    configMaps: {}
+    deployments: {}
+    webhookConfigurationOptions:
+      validation.webhook.pipeline.tekton.dev:
+        failurePolicy: "Ignore"
+        timeoutSeconds: 20
+        SideEffects: None
```

Comment:
please remove the changes from `TektonPipeline.md`

Reply:
done